### ISSUE TITLE
OwinStartupAttribute.FriendlyName value 'UmbracoDefaultOwinStartup' does not match the given value 'UmbracoIdentityStartup' in Assembly 'umbraco, Version=1.0.6974.19963

### ISSUE THREAD

**Issue by cdecinkoKnight** [Shazwazza/UmbracoIdentity#97]
Fresh install of Umbraco 7.13.2, very first package to be installed.  Refresh site and get this nasty error:

```
- The OwinStartupAttribute.FriendlyName value 'UmbracoDefaultOwinStartup' does not match the given value 'UmbracoIdentityStartup' in Assembly 'umbraco, Version=1.0.6974.19963, Culture=neutral, PublicKeyToken=null'.
- The given type or method 'UmbracoIdentityStartup' was not found. Try specifying the Assembly.
To disable OWIN startup discovery, add the appSetting owin:AutomaticAppStartup with a value of "false" in your web.config.
To specify the OWIN startup Assembly, Class, or Method, add the appSetting owin:AppStartup with the fully qualified startup class or configuration method name in your web.config.
```

**Comment by Shazwazza** [comment]
I cannot replicate. Have performed the same steps here https://github.com/Shazwazza/UmbracoIdentity/issues/98#issuecomment-499700221 , perhaps my questions at the bottom are the same problem you are having, potential namespace conflicts or something.

**Comment by medarke** [comment]
I already have a Umbraco installation, when I installed the nuget package, rebuild I get this:

The following errors occurred while attempting to load the app.
- The OwinStartupAttribute.FriendlyName value 'UmbracoCustomOwinStartup' does not match the given value 'UmbracoIdentityStartup' in Assembly 'Omobono.CaseStudies.Umbraco, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
- The OwinStartupAttribute.FriendlyName value 'UmbracoStandardOwinStartup' does not match the given value 'UmbracoIdentityStartup' in Assembly 'Omobono.CaseStudies.Umbraco, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
- The OwinStartupAttribute.FriendlyName value 'UmbracoDefaultOwinStartup' does not match the given value 'UmbracoIdentityStartup' in Assembly 'Umbraco.Web, Version=8.0.0.0, Culture=neutral, PublicKeyToken=null'.
- The given type or method 'UmbracoIdentityStartup' was not found. Try specifying the Assembly.
To disable OWIN startup discovery, add the appSetting owin:AutomaticAppStartup with a value of "false" in your web.config.
To specify the OWIN startup Assembly, Class, or Method, add the appSetting owin:AppStartup with the fully qualified startup class or configuration method name in your web.config.

First time I have used this so maybe its something I'm doing wrong
